lib/v2/components/amount_entry/interactor/viewmodels/amount_entry_events.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:equatable/equatable.dart';
+import 'package:meta/meta.dart';
+
+/// --- EVENTS
+@immutable
+abstract class AmountEntryEvent extends Equatable {
+  @override
+  List<Object> get props => [];
+}
+
+class InitSendDataArguments extends AmountEntryEvent {
+  @override
+  String toString() => 'InitAmountEntryDataArguments: { InitAmountEntryDataArguments: }';
```

Comment:
The correct pattern in this is print the event name and if the event has a value and it is not sensitive information it is printed. 
More than anything, because this appears in the debugging logs, you don't want to see the event name repeated twice :)